### DISCUSSION TITLE
[Question]  the "markers" new feature and the current released version on github

### DISCUSSION THREAD

**Discussion by m040601** [asciinema/discussions#218] [Q&A]
First of all thank you for all the work being done on this remarkable and underrated tool  and ecosystem. I've been using it for many years. Nice to see it getting well maintained and small usefull features being introduced.

This is just a simple question and report about what is currently displayed on the README frontpage and what is read on the README shipped with the last released version.

I have a github "notifications" watch on this project. So today I got an alert about something about "markers". What ? I had never heard about that, is this something new, or is it something I missed in the past I asked my self.

This got me confused. Had I missed this before ? Did I fail to read something **_in my local installation_**

So I checked my local man pages and grepped my  /usr/share/doc/asciinema/README.md on Archlinux.

```
Name            : asciinema
Version         : 2.2.0-3
Description     : Record and share terminal sessions
Architecture    : any
URL             : https://asciinema.org/
Licenses        : GPL3
Groups          : None
Provides        : None
Depends On      : python
Optional Deps   : None
Required By     : None
Optional For    : None
Conflicts With  : None
Replaces        : None
Installed Size  : 424.93 KiB
Packager        : Felix Yan <felixonmars@archlinux.org>
Build Date      : Thu 06 Apr 2023 06:34:16 PM WEST
Install Date    : Wed 03 May 2023 10:25:08 PM WEST
Install Reason  : Explicitly installed
Install Script  : No
Validated By    : Signature
```

Hmm  the package is fresh and even rebuilt up to date. It seems to build from the last release in 2022 from github. But no mention of "markers".

So I went to the frontpage README and indeed, they are there very well described.

I then checked 

```
git diff b86f7cb -- README.md
```

And indeed this seems to be some README update very fresh from some weeks ago. Witho work being done on marker from some time ago.

But it's already there "advertised" on the README, of the main branch of https://github.com/asciinema/asciinema/, so I assumed it was "already shipped". But it's not because the last released version is from 2022.

So, sorry for the long prose, is there a plan to release a new version on github ?

**Comment by ku1ik** [comment]
Thanks for kind words, I'm glad you like the project :)

Markers support has been released in the player: https://github.com/orgs/asciinema/discussions/216
This can be used already if you self-host the player, or by configuring marker in recording's settings form on asciinema.org
Support for inserting markers during recording session has been added to the recorder (this repo) but hasn't been released yet.
